README.md

```diff
@@ -0,0 +1,97 @@
+[![Dependency Status](https://gemnasium.com/badges/github.com/Integreat/integreat-app.svg)](https://gemnasium.com/github.com/Integreat/integreat-app)
+versioning: ![versioning](https://img.shields.io/badge/calver-YYYY.MM.PATCH-22bfda.svg)
+
+# integreat-app
+## Contents
+* [Contributing](#contributing)
+* [Getting Started](#getting-started)
+* [Bug Reporting](#bug-reporting)
+* [Pull Requests](#pull-requests)
```

Comment:
Not sure if this has to be mentioned here separately as it is already part of contributing.